web/styles/modal.css

```diff
@@ -157,6 +157,22 @@
     }
 }
 
+#user-profile-modal .save-success {
+    display: inline-block;
+    vertical-align: top;
+    height: auto;
```

Comment:
Do we need to set the values of `height` and `width`? I guess by default their value is set to `auto`.

Reply:
Yes we don't need the `display` too i think.